inst/include/interface/rcpp/rcpp_objects/rcpp_maturity.hpp

```diff
@@ -78,12 +79,85 @@ class LogisticMaturityInterface : public MaturityInterfaceBase {
   virtual double evaluate(double x) {
     fims_popdy::LogisticMaturity<double> LogisticMat;
     LogisticMat.inflection_point.resize(1);
```

Comment:
now that this parameter is time varying, we need to remove the code that fixes this vector to a size of 1